src/test/ui/expr/malformed_closure/missing_braces_around_block.stderr

```diff
@@ -0,0 +1,38 @@
+error: closure bodies that contain statements must be surrounded by braces
+  --> $DIR/missing_braces_around_block.rs:14:26
+   |
+LL |     (1..num).reduce(|a, b|
+   |                          ^
+...
+LL |     ).unwrap();
+   |     ^
+   |
+note: statement found outside of a block
+  --> $DIR/missing_braces_around_block.rs:16:26
+   |
+LL |         println!("{}", a);
+   |         -----------------^ this `;` turns the preceding expression into a statement
+   |         |
+   |         this expression is a statement because of the trailing semicolon
```

Comment:
Rereading with fresh eyes, I think that this is slightly misleading: the `;` turns the previous _closure_ into a statement, as far as the parser is concerned, but I feel it might be a reasonable "white lie" to not confuse people _too_ much.